latest/src/app/documentation/demos/angular-grid/angular-grid.demo.html

```diff
@@ -0,0 +1,41 @@
+<clr-doc-wrapper [title]="title">
+    <article>
+        <div id="design-guidelines">
+            <h3>Max Width</h3>
+            <p>The max width which defines the maximum width of an element can be set with the <code class="clr-code">clr-max-width</code>.<br>
```

Comment:
please rephrase this sentence and mention that it's a css class. Additionally provide style guide description when to use this class